cljfmt/src/cljfmt/core.cljc

```diff
@@ -248,8 +248,30 @@
 (defn- reader-conditional? [zloc]
   (and (reader-macro? zloc) (#{"?" "?@"} (-> zloc z/down token-value str))))
 
+(defn- reader-conditional-form-symbol
+  "Given a zipper pointing to a reader conditional form like
+
+    #?(:clj potemkin.core/defprotocol+ :cljs defprotocol)
+
+  Return the first symbol inside it e.g.
+
+    potemkin.core/defprotocol+"
```

Comment:
Can you remove this docstring? Since this is a private function, we don't need to give it public documentation. Instead, perhaps improve the function name. Something like: `first-symbol-in-reader-conditional`.